src/http/interceptable-http-proxy.service.ts

```diff
@@ -25,8 +25,10 @@ export class InterceptableHttpProxyService implements ProxyHandler<any> {
   apply(target: any, thisArg: any, argArray?: any): any {
     const method = InterceptableHttpProxyService._callStack.pop();
 
+    const context = {};
```

Comment:
It is better to use Object without prototype (`Object.create(null)`) for hash maps so it will not have some standard properties like `toString` on the context.